tests/ctst/package.json

```diff
@@ -21,7 +21,7 @@
     "@typescript-eslint/eslint-plugin": "^5.45.0",
     "@typescript-eslint/parser": "^5.45.0",
     "babel-jest": "^29.3.1",
-    "cli-testing": "github:scality/cli-testing.git#0.2.15",
+    "cli-testing": "github:scality/cli-testing.git#b4c831fd63896c32599451e063895042c96c721e",
```

Comment:
Adding a comment here as a reminder to switch to the version rather than the hash once this PR is going to be merged. Otherwise LGTM